molecule/umd5/molecule.yml

```diff
@@ -7,11 +7,12 @@ driver:
   name: docker
 scenario:
   name: umd5
-lint: |
-  set -e
-  yamllint .
-  ansible-lint --force-color
-  flake8 --config .github/linters/.flake8
+# Not needed since we have this in the pre-commit hooks
```

Comment:
People many fail to setup the pre-commit stuff, I feel it may still be useful to have the lint checks be formally done as part of the testing pipeline.
Or if you think it's really a duplication/waste of time, just remove the code instead of commenting.